src/cartridges/app_adyen_SFRA/cartridge/client/default/js/paypalExpress.js

```diff
@@ -0,0 +1,225 @@
+const {
+  getPaymentMethods,
+  updateLoadedExpressMethods,
+  checkIfExpressMethodsAreReady,
+} = require('./commons');
+const helpers = require('./adyen_checkout/helpers');
+const { PAYPAL } = require('./constants');
+
+async function callPaymentFromComponent(data, component) {
+  try {
+    $.spinner().start();
+    const response = await fetch(window.makeExpressPaymentsCall, {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify(data),
+    });
+    const { action, errorMessage = '' } = await response.json();
+    if (response.ok && action) {
+      component.handleAction(action);
+    } else {
+      throw new Error(errorMessage);
+    }
+  } catch (e) {
+    component.handleError();
```

Comment:
Are we ignoring the error message here?

Reply:
Yeah, we ignore it also for other payment methods. Ideally, we don't want to log on the front-end